docs/umi/examples.md

```diff
@@ -32,5 +32,10 @@ This is an example of an end-to-end cmd-line-only workflow:
     # Remove poly(A) tails and concatemer
     $ isoseq3 refine flt.bam primers.fasta fltnc.bam --require-polya
 
+    # Correct Barcodes
+    $ isoseq3 correct --barcodes include.txt fltnc.bam corrected.bam
+
+    $samtools sort -t CB corrected.bam -o corrected.sorted.bam
+
     # Deduplicate transcripts from the identical molecule
```

Comment:
we aren't deduplicating transcripts - we are deduplicating identical molecules (you have still end up w unique molecules representing the same transcript), hence change to

"# Deduplicate reads based on UMIs"